### DISCUSSION TITLE
state not updating

### DISCUSSION THREAD

**Discussion by lockcoin5518** [CodenameCrew/CodenameEngine#280] [Q&A]
i am trying to make a custom main menu and whenever i edit the MainMenuState.hx nothing changes in game. what am i doing wrong???

i have done as much digging as possible and i cant find an answer ANYWHERE

![image_2024-04-12_230607027](https://github.com/FNF-CNE-Devs/CodenameEngine/assets/161005326/f94486e2-4fdc-47ed-a834-01ae85326639)

**Comment by abandonedacc2123** [comment]
are you checking the console for errors n shit

**Reply by lockcoin5518** [reply]
no but i just checked the console and it just says 

[  23:35:06  |    ERROR    ] MainMenuState.hx:1: Unexpected token: "funkin"

and thanks, i didn't think to use the console in the debug option menu thingy

**Reply by abandonedacc2123** [reply]
show the code

**Reply by lockcoin5518** [reply]
![image_2024-04-13_153558490](https://github.com/FNF-CNE-Devs/CodenameEngine/assets/161005326/31632cfc-1a08-41af-a066-6f659424991e)
i haven't changed line one at all from the source code mainmenustate.hx

**Reply by NexIsDumb** [reply]
> ![image_2024-04-13_153558490](https://github.com/FNF-CNE-Devs/CodenameEngine/assets/161005326/31632cfc-1a08-41af-a066-6f659424991e)
> 
> i haven't changed line one at all from the source code mainmenustate.hx

delete that line
